filament/include/filament/MaterialInstance.h

```diff
@@ -234,6 +234,28 @@ class UTILS_PUBLIC MaterialInstance : public FilamentAPI {
         setParameter(name, strlen(name), type, color);
     }
 
+    /**
+     * Gets the value of a parameter by name.
+     * 
+     * @param name          Name of the parameter as defined by Material. Cannot be nullptr.
+     * @param nameLength    Length in `char` of the name parameter.
+     * @throws utils::PreConditionPanic if name doesn't exist or no-op if exceptions are disabled.
+     */
```

Comment:
you could also add one can check the paramter exists with `Material::hasParameter` and add a `@see` tag.

Reply:
Sure! Added the `@see` tag , but what do you mean by "can check"?

`getFieldInfo(name)` already has a `ASSERT_PRECONDITION` that checks if the info with the given name was found, so I thought that'd be sufficient.

Where would you like me to add the `hasParameter` check?

Reply:
I just meant we can add in the documentation that it's possible to check the that parameter exists using `hasParameter`. For some applications, it's not acceptable to fail, so they can check up-front.